docker-compose-dev.yml

```diff
@@ -19,6 +19,33 @@ services:
     ports:
       - "9000:9000"
 
+  initial-buckets:
+    image: minio/mc
+    depends_on:
+      - minio-dev
+    environment:
+      MINIO_ACCESS_KEY: minio
+      MINIO_SECRET_KEY: minio123
+    networks:
+      - xain-fl-dev
+    entrypoint: >
+      /bin/sh -c "
+        echo Waiting for minio service to start...;
+        while ! nc -z minio-dev 9000;
+        do
+          sleep 1;
```

Comment:
I often get:
```
initial-buckets_1  | mc: <ERROR> Unable to initialize new config from the provided credentials. Server not initialized, please try again.
```
Can we increase the sleep time a bit?

Reply:
This is literally waiting until minio is backup, the wait is to attempt to connect with netcat. 
This looks like a different problem

Reply:
confirmed that this is a timing issue, fixed and adding comments in the PR